templates/zig/src/tic80.zig

```diff
@@ -269,7 +269,7 @@ pub const trib = raw.trib;
 
 const TextriArgs = struct {
     texture_source : TextureSource = TextureSource.TILES,
-    transparent: []const u8 = .{},
+    transparent: []const u8 = &.{},
```

Comment:
This is a bit strange, what version of Zig were you using again?

Reply:
Just the latest master, since 0.9.1 is missing the `export_table` field on `LibExeObjStep`. 

Without the `&`, it seems to be an array literal (type omitted), which can't be coerced to a slice. The pointer to the array can be coerced though. ([Zig Documentation](https://ziglang.org/documentation/master/#Type-Coercion-Slices-Arrays-and-Pointers))